tests/ui/drop/lint-tail-expr-drop-order.rs

```diff
@@ -0,0 +1,33 @@
+//@compile-flags: -Z unstable-options
+//@edition:2024
```

Comment:
Nit: (consistency with other tests)

```suggestion
//@ compile-flags: -Z unstable-options
//@ edition: 2024
```

Reply:
Applied